runtime/domain.c

```diff
@@ -991,6 +991,11 @@ int caml_try_run_on_all_domains(void (*handler)(caml_domain_state*, void*, int,
   return caml_try_run_on_all_domains_with_spin_work(handler, data, leader_setup, 0, 0);
 }
 
+int caml_check_pending_actions() {
+  return (((uintnat)Caml_state->young_ptr - Bhsize_wosize(Max_young_wosize) <
```

Comment:
I don't understand this particular check. Can you explain how this check works?

Reply:
This function is actually left over from the EINTR combined PR. I've removed it for now.

(It's checking specifically whether the minor heap is full-ish)